payments-core/src/main/java/com/stripe/android/payments/core/analytics/ErrorReporter.kt

```diff
@@ -96,6 +96,9 @@ interface ErrorReporter {
         CUSTOMER_SHEET_PAYMENT_METHODS_LOAD_FAILURE(
             eventName = "elements.customer_sheet.payment_methods.load_failure"
         ),
+        CUSTOMER_SHEET_PAYMENT_METHODS_REFRESH_FAILURE(
```

Comment:
I think for these expected error events to be useful for alerting, you typically need to add a corresponding success event. Then, you can add an alert based on a percentage threshold

Reply:
Ah I actually didn't see that there was a `SuccessEvent` as well. I'd add one for this.